robot/Cumulus/tests/browser/crlps/last_soft_credit_opportunity.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/ContactPageObject.py
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/CustomizableRollupsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/ObjectMangerPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Validate And Create Required CustomField
+...             Enable Customizable Rollups
+Suite Teardown  Delete Records and Close Browser
+
+***Keywords***
+# creates test data a contact and an opportunity for the contact
+Setup Test Data
+    Setupdata   contact   ${contact1_fields}     ${opportunity_fields}
+    ${date} =   Get Current Date       result_format=%-m/%-d/%Y
+    Set suite variable    ${date}
+
+# Navigates to Object Manger for the specified object and checks for the presence of the custom field
+# If the specified custom field is not present adds the custom field
```

Comment:
instead of comments like this, u could add this as documentation for the keyword. Also could u move this to .robot file with arguments so that I could use same keyword for my test aswell